source/backend/tests/core/TestHelper.cs

```diff
@@ -11,8 +11,10 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
 using Moq;
+using Pims.Api.Services;
 using Pims.Dal;
 using Pims.Dal.Configuration.Generators;
+using Pims.Dal.Entities;
```

Comment:
this seems odd, looks like you are adding imports but then not using them anywhere?